alphadia/validation/schemas.py

```diff
@@ -317,37 +132,17 @@ def candidates_df(df: pd.DataFrame, logging: bool = True):
 
     Raises
     ------
-
     ValueError
         If validation fails
 
     """
     check_critical_values(df)
-    candidates_schema(df, logging=logging)
+    candidates_schema.validate(df, logging=logging)
 
 
 candidates_df.__doc__ += candidates_schema.docstring()
 
 
-def check_critical_values(input_df):
```

Comment:
I know it's not validation, but this was quite useful to figure out if something is wrong